src/Controller/PdfExportController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Controller;
+
+use App\Command\CommandDispatcherTrait;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use PhpOffice\PhpWord\PhpWord;
+use PhpOffice\PhpWord\Settings;
+use PhpOffice\PhpWord\Shared\Html;
+
+class PdfExportController extends Controller
+{
+    use CommandDispatcherTrait;
+    /**
+     * @Route("/cfdoc/{id}/pdf", name="export_pdf_file")
+     * @Method("GET")
+     *
+     * @param int $id
+     */
+    public function exportPdfAction(int $id, Request $request)
+    {
+        $phpWordObject = $this->get('phpword')->createPHPWordObject();
```

Comment:
Should use dependency injection to the constructor or action over using `get` from the container.

But in this case, `new PhpWord()` is all that is really needed and is clearer.